src/templates/keep-informed/hmrc-2015/index.hbs

```diff
@@ -0,0 +1,35 @@
+{{#extend "layout"}}
+
+  {{#replace "meta"}}
+    <title>Keep informed about legislation changes | FinSecPTX</title>
```

Comment:
So `review-hmrc-april-2015` might be better URL and "HMRC April 2015 review" as page title would be good.